models/Asset.php

```diff
@@ -1772,19 +1772,21 @@ public function getMetadata($name = null, $language = null, $strictMatch = false
         \Pimcore::getEventDispatcher()->dispatch($preEvent, AssetEvents::PRE_GET_METADATA);
         $this->metadata = $preEvent->getArgument('metadata');
 
-        $convert = function ($metaData) {
-            $loader = \Pimcore::getContainer()->get('pimcore.implementation_loader.asset.metadata.data');
-            $transformedData = $metaData['data'];
+        if (!$raw) {
+            $convert = function ($metaData) {
```

Comment:
It's because the declaration is done within this `if` ... but `$convert` is used further down outside this `if`. 
I also don't understand what this change should improve ... it's just the function declaration that's not done if `$raw = true`. 🤔